src/Foundatio/Jobs/IJob.cs

```diff
@@ -63,11 +63,11 @@ public static async Task<int> RunContinuousAsync(this IJob job, TimeSpan? interv
 
             if (result.Error != null)
             {
-                await SystemClock.SleepSafeAsync(Math.Max((int)(interval?.TotalMilliseconds ?? 0), 100), cancellationToken).AnyContext();
+                await job.GetTimeProvider().SafeDelay(TimeSpan.FromMilliseconds(Math.Max((int)(interval?.TotalMilliseconds ?? 0), 100)), cancellationToken).AnyContext();
```

Comment:
SafeDelayAsync?

Reply:
No, it's named after the TimeProviders Delay method.